sktime/tests/test_resizing.py

```diff
@@ -0,0 +1,30 @@
+import numpy as np
+import pandas as pd
+
+from sktime.datasets import load_basic_motions
+from sktime.transformers.resizing import TSResizeTransform
+
+# load data
+df, _ = load_basic_motions(split='TRAIN', return_X_y=True)
```

Comment:
please rename df to X as is standard for features

Reply:
done